spec/features/check_box_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "single check boxes" do
+  before do
+    visit element_path("check_box")
+  end
+
+  it "renders the visible unchecked checkbox and label" do
+    expect(page).to have_unchecked_field("person[opted_in]", type: "checkbox", with: "1")
+  end
+
+  it "renders the hidden field which defaults to false" do
+    expect(page).to have_field("person[opted_in]", type: "hidden", with: "0")
+  end
+
+  it "renders the visible checked checkbox and label" do
```

Comment:
The description mentions the label, however the test body doesn't check for one. You can target the checkbox via the label text with:

```ruby
expect(page).to have_checked_field("Account active", type: "checkbox", with: "1")
```